FusionIIIT/applications/otheracademic/views.py

```diff
@@ -170,65 +416,1587 @@ def upload_file(request, entry_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.bank_clear= True
+    leave_entry.bank_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Bank_nodues')  # Red   
+
+def approve_bank_not(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.bank_clear= True
+    leave_entry.bank_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Bank_nodues_not')  # Red    
+    
+
+
+def reject_bank(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.bank_clear= False
+    leave_entry.bank_notclear = True
+    leave_entry.save()
+    return redirect('/otheracademic/Bank_nodues')  # Red    
+
+def approve_CSE(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.cc_clear= True
+    leave_entry.cc_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/CSE_nodues')  # Red    
+
+def approve_CSE_not(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.cc_clear= True
+    leave_entry.cc_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/CSE_nodues_not')  # Red  
+    
+
+
+def reject_CSE(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.cc_clear= False
+    leave_entry.cc_notclear = True
+    leave_entry.save()
+    return redirect('/otheracademic/CSE_nodues')  # Red  
+
+def approve_design_project(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.design_project_clear= True
+    leave_entry.design_project_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Design_nodues')  # Red    
+
+def approve_design_project_not(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.design_project_clear= True
+    leave_entry.design_project_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Design_nodues_not')  # Red 
+    
+
+
+def reject_design_project(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.design_project_clear= False
+    leave_entry.design_project_notclear = True
+    leave_entry.save()
+    return redirect('/otheracademic/Design_nodues')  # Red      
+
+def approve_design_studio(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.design_studio_clear= True
+    leave_entry.design_studio_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Design_nodues')  # Red  
+
+def approve_design_studio_not(request, no_dues_id):
+    leave_entry = NoDues.objects.get(id=no_dues_id)
+    leave_entry.design_studio_clear= True
+    leave_entry.design_studio_notclear = False
+    leave_entry.save()
+    return redirect('/otheracademic/Design_nodues_not')  # Red  
+
+def reject_design_studio(request, no_dues_id):
```

Comment:
what is the difference between this function and the function just above

Reply:
There is mistake by myside while making video , i forgot to show this functionality .There is functionality that if any actor of no dues approval reject the no dues status ,the it forwarded to nodues_not , so that if next time when issue is cleared by student then actor able to clear the dues in form again.

Reply:
I resolved all your above query.Please review once.